service/controller/v25/resource/tccp/resource.go

```diff
@@ -33,12 +34,15 @@ type AWSConfig struct {
 // Config represents the configuration used to create a new cloudformation
 // resource.
 type Config struct {
-	APIWhitelist         adapter.APIWhitelist
+	APIWhitelist adapter.APIWhitelist
+	// EncrypterRoleManager manages role encryption. This can be supported by
+	// different implementations and thus is optional.
```

Comment:
I noticed this when playing around. I added validation for it but realized that there is not always a role manager given apparently. Further this seems to be some China specialty.